WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteCredentialsViewController.swift

```diff
@@ -0,0 +1,129 @@
+
+    // MARK: - Actions
+    @IBAction func handleContinueButtonTapped(_ sender: NUXButton) {
+
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+		localizePrimaryButton()
+		registerTableViewCells()
+		loadRows()
+		configureSubmitButton(animating: false)
+    }
+
+	// MARK: - Overrides
+
+    /// Style individual ViewController backgrounds, for now.
+    ///
+    override func styleBackground() {
+        guard let unifiedBackgroundColor = WordPressAuthenticator.shared.unifiedStyle?.viewControllerBackgroundColor else {
+            super.styleBackground()
+            return
+        }
+
+        view.backgroundColor = unifiedBackgroundColor
+    }
+}
+
+
+// MARK: - UITableViewDataSource
+extension SiteCredentialsViewController: UITableViewDataSource {
+    /// Returns the number of rows in a section.
+    ///
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return rows.count
+    }
+
+    /// Configure cells delegate method.
+    ///
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let row = rows[indexPath.row]
+        let cell = tableView.dequeueReusableCell(withIdentifier: row.reuseIdentifier, for: indexPath)
+        configure(cell, for: row, at: indexPath)
+
+        return cell
+    }
+}
+
+
+// MARK: - Private Methods
+private extension SiteCredentialsViewController {
+
+	/// Localize the "Continue" button.
+    ///
+    func localizePrimaryButton() {
+        let primaryTitle = WordPressAuthenticator.shared.displayStrings.continueButtonTitle
+        submitButton?.setTitle(primaryTitle, for: .normal)
+        submitButton?.setTitle(primaryTitle, for: .highlighted)
+    }
+
+	/// Registers all of the available TableViewCells.
+    ///
+    func registerTableViewCells() {
+        let cells = [
+            TextLabelTableViewCell.reuseIdentifier: TextLabelTableViewCell.loadNib()
+        ]
+
+        for (reuseIdentifier, nib) in cells {
+            tableView.register(nib, forCellReuseIdentifier: reuseIdentifier)
+        }
+    }
+
+	/// Describes how the tableView rows should be rendered.
+    ///
+    func loadRows() {
+        rows = [.instructions]
+    }
+
+	/// Configure cells.
+    ///
+    func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
+        switch cell {
+        case let cell as TextLabelTableViewCell where row == .instructions:
+            configureInstructionLabel(cell)
+        default:
+            DDLogError("Error: Unidentified tableViewCell type found.")
+        }
+    }
+
+	/// Configure the instruction cell.
+    ///
+    func configureInstructionLabel(_ cell: TextLabelTableViewCell) {
+        cell.configureLabel(text: "Enter your account information for pamelanguyen.com.", style: .body)
+    }
+
+
+	// MARK: - Private Constants
+
+    /// Rows listed in the order they were created.
+    ///
+    enum Row {
+        case instructions
```

Comment:
just a thought: unless you are going to have a considerable number of different cases, maybe you could just refer to the constant (`TextLabelTableViewCell.reuseIdentifier`) since it's already accessible from the same scope?

Reply:
There are going to be 5 cases in total, so this enum will expand quite a bit in the next PR. I put the instruction row in so that it proves the table is displaying correctly and populating data.